revolv/payments/models.py

```diff
@@ -23,63 +36,92 @@ class PaymentInstrumentType(models.Model):
     objects = PaymentInstrumentTypeManager()
 
 
-class PaymentTransactionManager(models.Manager):
+class PaymentManager(models.Manager):
+    """
+        Simple manager for the Payment model.
+    """
+
+    def payments(self, user=None, project=None, queryset=None):
+        """
+        :return: Payments associated with this user
+        """
+        if queryset is None:
+            queryset = super(PaymentManager, self).get_queryset()
+        payments = queryset.filter(user=user).order_by('created_at')
+        return payments
+
+    def donations(self, user=None, project=None, queryset=None):
+        """
+        :return: Returns all payments that are not repayments or reinvestments
+                 associated with this user.
+        """
+        if queryset is None:
+            queryset = super(PaymentManager, self).get_queryset()
```

Comment:
Do you think it's cleaner to write:

    queryset = queryset or super(PaymentManager, self).get_queryset()